<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_part_3_2</class>
 <widget class="QDialog" name="fct_part_3_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sportifs d'un pays et un Ã©quipe</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../342px-Olympic_rings_without_rims.svg.png</normaloff>../../../../../../../342px-Olympic_rings_without_rims.svg.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>NumEp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_part_3_2_numEp"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>||</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Or</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_part_3_2_or"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Argent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_part_3_2_argent"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Bronze</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_part_3_2_bronze"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_part_3_2">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_part_3_2_numEp</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_part_3_2</receiver>
   <slot>initialiserInscrits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_part_3_2</receiver>
   <slot>insertResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initialiserInscrits()</slot>
  <slot>insertResult()</slot>
 </slots>
</ui>
